### PR TITLE
`std::endl` to `'\n'`

### PR DESCRIPTION
**Pull request by servusDei2018**
Changed prominent occurrences of `std::endl` to `'\n'`for speed boost. `endl` includes unnecessary `std::flush`. Benchmarks show improved performance.

### CODE REVIEW

**Comment by GPUtester** [comment]
Can one of the admins verify this patch?

**Comment by servusDei2018** [comment]
There are more occurrences but these are just the most prominent ones.

File: io/src/Library/Logging/CoutOutput.cpp [hunk 1]
```diff
@@ -10,12 +10,12 @@ CoutOutput::~CoutOutput() {}
 
 void CoutOutput::flush(std::string && log) {
 	std::unique_lock<std::mutex> lock(mutex);
-	std::cout << log << std::endl;
+	std::cout << log << '\n';
```
Comment (felipeblazing) [inline io/src/Library/Logging/CoutOutput.cpp:13]: the point of this function is to flush. It absolutely wants to use std::endl

File: io/src/Library/Logging/FileOutput.cpp [hunk 2]
```diff
@@ -29,19 +29,19 @@ FileOutput::~FileOutput() {}
 
 void FileOutput::flush(std::string && log) {
 	std::unique_lock<std::mutex> lock(mutex);
-	file << log << std::endl;
+	file << log << '\n';
```
Comment (felipeblazing) [inline io/src/Library/Logging/FileOutput.cpp:32]: this function is meant to flush.

File: io/src/Library/Logging/FileOutput.cpp [hunk 3]
```diff
@@ -29,19 +29,19 @@ FileOutput::~FileOutput() {}
...
 }
 
 void FileOutput::flush(const std::string & log) {
 	std::unique_lock<std::mutex> lock(mutex);
-	file << log << std::endl;
+	file << log << '\n';
```
Comment (felipeblazing) [inline io/src/Library/Logging/FileOutput.cpp:37]: thiose one is meant to flush as well

File: io/src/Library/Logging/FileOutput.cpp [hunk 4]
```diff
@@ -29,19 +29,19 @@ FileOutput::~FileOutput() {}
...
 }
 
 
 void FileOutput::flush(
 	const int nodeInd, const std::string & datetime, const std::string & level, const std::string & log) {
 	std::unique_lock<std::mutex> lock(mutex);
-	file << datetime << "|" << nodeInd << "|" << level << "|" << log << std::endl;
+	file << datetime << "|" << nodeInd << "|" << level << "|" << log << '\n';
```
Comment (felipeblazing) [inline io/src/Library/Logging/FileOutput.cpp:44]: another flusher!

File: comms/src/blazingdb/transport/Client.cc [hunk 5]
```diff
@@ -61,7 +61,7 @@ class ConcreteClientTCP : public ClientTCP {
     zmq::message_t local_message;
     auto success = socket_ptr->recv(local_message);
     if (success.value() == false || local_message.size() == 0) {
-      std::cerr << "Client:   throw zmq::error_t()" << std::endl;
+      std::cerr << "Client:   throw zmq::error_t()" << '\n';
```
Comment (felipeblazing) [inline comms/src/blazingdb/transport/Client.cc:64]: If the concern is the perf cost of the flush. we might as well make this 
```

std::cerr << "Client:   throw zmq::error_t()\n;
```

File: comms/src/blazingdb/transport/Server.cc [hunk 6]
```diff
@@ -216,7 +216,7 @@ void ServerTCP::Run() {
 					this->putMessage(message->metadata().contextToken, message);
 				}
 			} catch(const std::runtime_error & exception) {
-				std::cerr << "[ERROR] " << exception.what() << std::endl;
+				std::cerr << "[ERROR] " << exception.what() << '\n';
```
Comment (felipeblazing) [inline comms/src/blazingdb/transport/Server.cc:219]: cerr doesnt get buffered so I am not sure removing the flush here actually helps.

**Comment by felipeblazing** [comment]
Some of the comments were resolved but the issues don't seem to be addressed. The functions that are called flush() are meant to flush the log.

**Comment by williamBlazing** [comment]
ok to test

**Pull request closed without merging**